### PR TITLE
[#65] Supported object Content-Type

### LINKED ISSUE #65: Support "Content-Type" well-known attribute
We're autodetecting Content-Type header at the moment even though we have well-known `AttributeContentType` defined for objects. HTTP gate should take and use that if it's defined for an object (and fallback to autodetection otherwise) the same way it does for `FileName` and `Timestamp` attributes at the moment.

### PR DESCRIPTION
**Pull request by KirillovDenis**
closes #65 

Signed-off-by: Denis Kirillov <denis@nspcc.ru>

### CODE REVIEW

File: downloader/download.go [hunk 1]
```diff
@@ -141,13 +123,19 @@ func (r *request) receiveFile(clnt client.Object, objectAddress *object.Address)
...
 		}
 	}
 	r.Response.Header.Set("x-object-id", obj.ID().String())
 	r.Response.Header.Set("x-owner-id", obj.OwnerID().String())
 	r.Response.Header.Set("x-container-id", obj.ContainerID().String())
-	r.SetContentType(writer.contentType)
 	r.Response.Header.Set("Content-Disposition", dis+"; filename="+path.Base(filename))
+
+	if contentType == "" {
+		contentType = http.DetectContentType(r.Response.Body())
```
Comment (roman-khimov) [inline downloader/download.go:136]: It makes the server read full object body and pass it to detection function while we only need 512 bytes of it. Think of huge objects, we better be streaming them as we read from NeoFS node.

**Review by roman-khimov — APPROVED** [review]
OK, it'll work as of now and we'll deal with proper streaming later.

**Comment by realloc** [comment]
What about content type detection when the attribute is not set? Does it happen automatically with fasthttp?

**Comment by roman-khimov** [comment]
It happens in the gateway at the moment and NeoFS metadata overrides this detection.

**Pull request merged**